cbcontainers/communication/gateway/api_gateway.go

```diff
@@ -90,17 +91,33 @@ func (gateway *ApiGateway) baseRequestWithRetries() *resty.Request {
 }
 
 func (gateway *ApiGateway) getResourcePathWithAccountPath(resourceName string) string {
-	return gateway.baseUrl(fmt.Sprintf("account/%s/%s", gateway.account, resourceName))
+	return gateway.baseUrl(fmt.Sprintf("management/%v", resourceName))
 }
 
-func (gateway *ApiGateway) RegisterCluster() error {
+func (gateway *ApiGateway) SplitToGroupAndMember() (string, string, error) {
```

Comment:
@yurado wanted to update the API

Reply:
The operator currently used the deprecated API. @yurado Asked me to update the operator to use the new API rather than the deprecate one during this task.